force-app/main/default/classes/tasks/CollectionsTasks.cls

```diff
@@ -26,6 +45,29 @@ public class CollectionsTasks {
     Integer sum,
     List<Integer> listWithElementsToCheck
   ) {
+
+    listWithElementsToCheck.sort();
```

Comment:
In the task description there is a hint "arbitrary collection" - what other non-list collections can help you?
Discuss within a group to find a solution

Reply:
Updated. But still think my solutions was pretty enough :)